system/Helpers/kint_helper.php

```diff
@@ -51,13 +67,23 @@ function d(...$vars)
         /**
          * Provides a backtrace to the current execution point, from Kint.
          */
+        /**
+         * trace function
+         *
+         * @return int|string
```

Comment:
This method return nothing.
```suggestion
         * @return void
```

Reply:
OK. Tks for review